lib/kitchen/command.rb

```diff
@@ -170,17 +170,25 @@ def run_action(action, instances, *args)
         concurrency.times do
           threads << Thread.new do
             while instance = queue.pop
+              puts "running #{instance.name}"
```

Comment:
I had the same question and then noticed `puts` getting used for these in several places in TK. a bit weird but separate from this PR